openedx/features/course_experience/static/course_experience/fixtures/course-home-fragment.html

```diff
@@ -67,19 +67,19 @@ <h2 class="hd hd-3 page-title">Reviews Test Course</h2>
                     <h3 class="hd-6">Course Tools</h3>
                     <ul class="list-unstyled">
                         <li>
-                            <a class="course-tool-link" href="/courses/course-v1:W3Cx+HTML5.0x+1T2017/bookmarks/">
+                            <a class="course-tool-link" data-analytics_id="bookmarks" href="/courses/course-v1:W3Cx+HTML5.0x+1T2017/bookmarks/">
```

Comment:
I imagine that "data-analytics-id" would be a more consistent naming convention.  I'd also add the `edx.` into the fixture just to make the sample more complete.